src/opera/pge/base/schema/iso_metadata_measured_parameters_config_schema.yaml

```diff
@@ -0,0 +1,7 @@
+map(include('parameter'), key=str())
+---
+parameter:
```

Comment:
One nice thing about yaml schemas is that, unlike JSON, you can provide inline documentation. It would be nice to see each of these parameters documented with things like what the fallback behavior is when an optional field is omitted. We can also add links to the ISO schema to justify why the enums are defined as they are.